### ISSUE TITLE
Кнопка ответ больше не нужна, т.к. функционал заменил Быстрый ответ

### ISSUE THREAD

**Issue by Omny** [zikkuratvk/joomlaforum#31]
Раз появился полнофункциональный быстрый ответ внизу темы, то кнопка Ответ больше не нужна
![screenshot_1](https://user-images.githubusercontent.com/2302730/29492923-a13da5b0-8593-11e7-9e19-17d84fc51a03.jpg)

**Comment by Septdir** [comment]
Не согласен кнопка ответ ведет на полный редактор. А вот сделать ее другми цветом стоит

**Comment by seoelle** [comment]
В быстром ответе нельзя добавить вложение

**Comment by Omny** [comment]
Можно добавить кнопку вставки вложения и не усложнять людям жизнь переходами на отдельный редактор на отдельной странице

**Comment by zikkuratvk** [comment]
Возможно и можно было бы это доработать но не сейчас на далекую перспективу может быть и сделаем.
